src/net/slashie/expedition/domain/ItemOffshoreExpeditionContainer.java

```diff
@@ -118,8 +118,9 @@ public String getTypeDescription() {
 	}
 
 	@Override
-	public int getWaterDays() {
-		return getExpedition().getWaterDays();
+	public int getWaterDays() 
+	{		
+		return getExpedition().getOffShoreWaterDays();

```

Comment:
Will this work while the expedition is on sea? when will we use getExpedition().getWaterDays ?